### PR TITLE
Add in solution for YouTube tracker

### PR DESCRIPTION
**Pull request by aliciawilliams**

### CODE REVIEW

File: youtube-tracker/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Track YouTube video views and comments
+
+_Tracks views, likes, and comments for provided YouTube URLs in a Google Sheet with optional email notifications._
+
+Last updated: August, 2019
+
+Track performance of public YouTube videos, including views, likes, and comments, in a Google Sheet.
```
Comment (erickoledadevrel) [inline youtube-tracker/README.md]: This description feels very mechanical. Consider weaving a bit more story into this, talk about the problem being solved, etc. Doesn't need to be your exact story for creating it, but something that paints a better picture.

Some examples:
https://developers.google.com/gsuite/solutions/vacation-calendar
https://developers.google.com/gsuite/solutions/call-in-sick

File: youtube-tracker/README.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+Last updated: August, 2019
+
+Track performance of public YouTube videos, including views, likes, and comments, in a Google Sheet. 
+Refresh data automatically via the YouTube Data API. Send automatic email notifications
+that summarize new YouTube video comment activity.
+
+![screenshot](https://cdn.jsdelivr.net/gh/gsuitedevs/solutions@master/youtube-tracker/screenshot.png)
+
+## Technology highlights
+- Use the [YouTube advanced service](https://developers.google.com/apps-script/advanced/youtube) to collect data on specific videos (such as number of views and comment data) via the YouTube Data API.
```
Comment (erickoledadevrel) [inline youtube-tracker/README.md]: I think "Uses" would be the correct tense.

File: youtube-tracker/README.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+- Use the [Gmail Service](https://developers.google.com/apps-script/reference/gmail/) to send notification emails when new video comments are detected. 
+- Create an [HTML template](https://developers.google.com/apps-script/guides/html/templates#calling_apps_script_functions_from_a_template) containing an Apps Script function to send emails containing HTML tables.
```
Comment (erickoledadevrel) [inline youtube-tracker/README.md]: Ditto, "Creates"

Comment (erickoledadevrel) [inline youtube-tracker/README.md]: Using the word "containing" twice in the same sentence sounds weird. Consider rewording, perhaps being less descriptive.

File: youtube-tracker/README.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+## Try it
+
+### Spreadsheet setup
+1. Make of copy of the spreadsheet [here](https://docs.google.com/spreadsheets/d/12rQe1ndU_VmmHl0QIqUi-XxQ8lWovjh0xfOHTfxOHoo/copy). It already contains the Apps Script code from this repository.
```
Comment (erickoledadevrel) [inline youtube-tracker/README.md]: > from this repository.

It's not actually in this repository. Perhaps remove that part of the sentence.

File: youtube-tracker/README.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+2. Change the name of the tab to the full email address where you’d like to receive email notifications.
```
Comment (erickoledadevrel) [inline youtube-tracker/README.md]: We usually refer to tabs as "sheets". So "name of the sheet".

File: youtube-tracker/README.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+
+### YouTube Advanced Service setup
+1. From the spreadsheet, open the script editor by selecting Tools > Script editor.
+2. In the script editor, select Resources > Advanced Google services.
+3. In the Advanced Google Service dialog that appears, click the on/off switch next to the YouTube Data API service.
+4. Click OK in the dialog.
+
+### Test the code
+1. From the script editor, choose `markVideos` from the select box in the toolbar, then click ▶.
+2. You should see the details added in columns C through H, and you will receive an email for any videos that have more than zero comments. When running the function in the future, you will only receive an email with videos whose comment count has increased since the last time the script was run.
```
Comment (erickoledadevrel) [inline youtube-tracker/README.md]: You should mention before this that the user will be asked to authorize the script.

File: youtube-tracker/README.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+## Try it
+
+### Spreadsheet setup
+1. Make of copy of the spreadsheet [here](https://docs.google.com/spreadsheets/d/12rQe1ndU_VmmHl0QIqUi-XxQ8lWovjh0xfOHTfxOHoo/copy). It already contains the Apps Script code from this repository.
+2. Change the name of the tab to the full email address where you’d like to receive email notifications. 
+3. Locate URLs of videos you would like to track and add them in column A below cell A1.
+
+### YouTube Advanced Service setup
+1. From the spreadsheet, open the script editor by selecting Tools > Script editor.
+2. In the script editor, select Resources > Advanced Google services.
```
Comment (erickoledadevrel) [inline youtube-tracker/README.md]: FYI, in markdown it's traditional to use `1.` for all bullets in a numbered list. The rendered will assign the correct number, and then you can reorder the list more easily later.

Comment (erickoledadevrel) [inline youtube-tracker/README.md]: Anytime you reference a UI element, such as "Resources > Advanced Google Services", you should make it bold, but surrounding it with two asterisks. Fix that here and in the rest of the instructions.

File: youtube-tracker/README.md (hunk 3 above)
Comment (aliciawilliams) [reply youtube-tracker/README.md]: How about...
Creates an [HTML template](https://developers.google.com/apps-script/guides/html/templates#calling_apps_script_functions_from_a_template) containing an Apps Script function to send custom notification emails.

File: youtube-tracker/README.md (hunk 2 above)
Comment (aliciawilliams) [reply youtube-tracker/README.md]: Done

File: youtube-tracker/README.md (hunk 3 above)
Comment (aliciawilliams) [reply youtube-tracker/README.md]: Done

File: youtube-tracker/README.md (hunk 4 above)
Comment (aliciawilliams) [reply youtube-tracker/README.md]: Done

File: youtube-tracker/README.md (hunk 5 above)
Comment (aliciawilliams) [reply youtube-tracker/README.md]: done

File: youtube-tracker/README.md (hunk 7 above)
Comment (aliciawilliams) [reply youtube-tracker/README.md]: Cool! And done

Comment (aliciawilliams) [reply youtube-tracker/README.md]: Ok, I tried to do this but may have added too many UI elements...let me know.

File: youtube-tracker/README.md (hunk 6 above)
Comment (aliciawilliams) [reply youtube-tracker/README.md]: Done - copied from the vacation calendar

File: youtube-tracker/README.md (hunk 1 above)
Comment (aliciawilliams) [reply youtube-tracker/README.md]: I am submitting the rest of the changes, will work on making this one!

File: youtube-tracker/README.md (hunk 3 above)
Comment (erickoledadevrel) [reply youtube-tracker/README.md]: Works for me.

File: youtube-tracker/README.md (hunk 1 above)
Comment (erickoledadevrel) [reply youtube-tracker/README.md]: The rest looks good, let me know when this is done.

Comment (aliciawilliams) [reply youtube-tracker/README.md]: Ok, I updated the short and long descriptions. Let me know what you think!

**Pull request merged**